src/ruby/yast/logger.rb

```diff
@@ -1,6 +1,10 @@
 require "yastx"
 require "yast/builtins"
 
+# FIXME: a workaround for https://github.com/yast/yast-ruby-bindings/issues/133
+require "enc/encdb.so"
+require "enc/trans/transdb.so"
```

Comment:
interesting, especially if you see that we already have there encdb hack...do you place it after or before?